app/code/Magento/Customer/Model/Session.php

```diff
@@ -354,10 +370,11 @@ public function setCustomerGroupId($id)
     }
 
     /**
-     * Get customer group id
-     * If customer is not logged in system, 'not logged in' group id will be returned
+     * Get customer group id. If customer is not logged in system, 'not logged in' group id will be returned.
      *
-     * @return int
+     * @return int|null
```

Comment:
I reviewed and didn't found case when there could be null. Could you point me?

Reply:
> Could you point me?

I can not point you to this particular case but PHPStorm points to app/code/Magento/Customer/Api/Data/CustomerInterface.php, [line 61](https://github.com/magento/magento2/blob/2.3-develop/app/code/Magento/Customer/Api/Data/CustomerInterface.php#L61) ;)

Reply:
**fixed**

Reply:
Thanks, @ihor-sviziev